cmd/helm/helm_test.go

```diff
@@ -151,3 +154,52 @@ func testChdir(t *testing.T, dir string) func() {
 	}
 	return func() { os.Chdir(old) }
 }
+
+func TestPluginExitCode(t *testing.T) {
```

Comment:
This verifies that the `helm` command can return a non-zero exit status other than 1 from a plugin, by "actually running a helm process".